components/builder-protocol/src/message/mod.rs

```diff
@@ -41,6 +42,7 @@ impl<'a, T: 'a + protobuf::Message> Message<'a, T> {
             Some("routesrv") => net::Protocol::RouteSrv,
             Some("sessionsrv") => net::Protocol::SessionSrv,
             Some("vault") => net::Protocol::VaultSrv,
+            Some("scheduler") => net::Protocol::Scheduler,
```

Comment:
Ah yes, the hidden places you need to extend to add a service the router can talk to... This can be fixed in the future ;)